examples/mongodb-container/src/test/java/org/testcontainers/containers/MongoDbContainer.java

```diff
@@ -24,7 +25,9 @@
 
     /**
      * Creates a new {@link MongoDbContainer} with the {@value DEFAULT_IMAGE_AND_TAG} image.
+     * @deprecated use {@link MongoDbContainer(DockerImageName)} instead
      */
+    @Deprecated
```

Comment:
a bit unrelated, but... don't we have a mongodb module now? :D

Reply:
Yeah, we should discuss nuking this example...